tests/functional_tests/t0_main/keras/keras_model_checkpoint_tf_2_4.yea

```diff
@@ -0,0 +1,16 @@
+id: 0.keras.modelcheckpoint.tf24
+tag:
+    shard: tf24
```

Comment:
You need to add this new shard to both tox.ini and Circle config.yml for the tests to be run.